src/main/java/drake/TodoLeftBlank.java

```diff
@@ -0,0 +1,12 @@
+package drake;
+
+/**
+ * Exception thrown when an attempt to create a todo task is made without specifying a description.
+ * This is a subclass of StringIndexOutOfBoundsException, which is typically thrown to indicate that
+ * an index is either negative or greater than the size of the string.
+ */
+public class TodoLeftBlank extends StringIndexOutOfBoundsException {
```

Comment:
A bit of nitpicking but would ToDoLeftBlankException be a better alternative for this exception class name to let it sound like an Exception?